src/Data/Characters/Venti/index.tsx

```diff
@@ -126,12 +126,12 @@ const dmgFormulas = {
     full7: unequal(condBurstAbsorption, undefined, sum(
       prod(dmgNode("atk", datamine.burst.baseDmg, "burst"), 20),
       prod(dmgNode("atk", datamine.burst.absorbDmg, "burst", { hit: { ele: condBurstAbsorption } }), 15),
-      prod(lookup(condBurstAbsorption, objectKeyMap(absorbableEle, ele => reactions.anemo[`${ele}Swirl`]), naught), 7)
+      prod(lookup(condBurstAbsorption, objectKeyMap(absorbableEle, ele => getReactions(elementKey)[`${ele}Swirl`]), naught), 7)
```

Comment:
Lets just do #618 here and remove the venti formula, since people will try to ducttape this into multi-config for venti...